packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -42,33 +45,71 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
+    
     @media screen and (max-width: $terra-tiny-breakpoint) {
+      left: -100vw;
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
+      left: calc(-1*var(--terra-consumer-nav-width, 320px));
```

Comment:
We should only have to move 1 container and all the children within it will move too....